src/Meilisearch/MeilisearchClient.cs

```diff
@@ -53,7 +53,7 @@ public MeilisearchClient(HttpClient client, string apiKey = default)
         /// <returns>Returns the Meilisearch version with commit and build version.</returns>
         public async Task<MeiliSearchVersion> GetVersionAsync(CancellationToken cancellationToken = default)
         {
-            var response = await this.http.GetAsync("/version", cancellationToken).ConfigureAwait(false);
+            var response = await _http.GetAsync("/version", cancellationToken).ConfigureAwait(false);
```

Comment:
So now `this.` is the default way to lookup for the references C# does?

Reply:
Most C# OSS library and asp.net core code looks like this.